spring-boot-examples/consumer-app/src/test/java/io/dapr/springboot/examples/consumer/DaprTestContainersConfig.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2025 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.springboot.examples.consumer;
+
+import io.dapr.testcontainers.Component;
+import io.dapr.testcontainers.DaprContainer;
+import io.dapr.testcontainers.DaprLogLevel;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.context.annotation.Bean;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.RabbitMQContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@TestConfiguration(proxyBeanMethods = false)
+public class DaprTestContainersConfig {
+
+  @Bean
+  public Network getDaprNetwork() {
+    Network defaultDaprNetwork = new Network() {
+      @Override
+      public String getId() {
+        return "dapr-network";
+      }
+
+      @Override
+      public void close() {
+
+      }
+
+      @Override
+      public Statement apply(Statement base, Description description) {
+        return null;
+      }
+    };
+
```

Comment:
To be honest I don't understand why we need to use DockerClient, if you could add a comment to explain that would help

Reply:
This is to check the docker network for reuse scenarios. @eddumelendez might be able to explain why this is needed. I think there is no other way to check if a docker network exist without using the client.